docs/css/docs.css

```diff
@@ -2199,6 +2247,66 @@ button:focus,
   box-shadow: 0 0 0 2px #003e50;
 }
 
+.table {
+  width: 100%;
+  max-width: 100%;
+  margin-bottom: 1rem;
+}
+
+.table-responsive {
+  display: block;
+  width: 100%;
+  overflow-x: auto;
+  -ms-overflow-style: -ms-autohiding-scrollbar;
+}
+
+.table.responsive-utilities {
+  border: 0;
+  margin-bottom: 3em;
+}
+
+.table.responsive-utilities > tbody > tr > td,
+.table.responsive-utilities > tbody > tr > th,
+.table.responsive-utilities > tfoot > tr > td,
+.table.responsive-utilities > tfoot > tr > th,
+.table.responsive-utilities > thead > tr > td,
+.table.responsive-utilities > thead > tr > th {
+  padding: .75rem;
+  vertical-align: top;
+  border: 1px solid #eceeef;
```

Comment:
I think this table works better without borders on cells. Can you remove this line please?

Reply:
Will be done :slightly_smiling_face: